thirdparty/ofqf/qoscserver.cpp

```diff
@@ -88,7 +88,7 @@ void QOscServer::readyRead()
         		            args += data[ i++ ];
                   i++; //move one byte more!
         			    if ( ! args.isEmpty() ) {
-        				        QList<QVariant> list;
+        				        QList<QVariant> list1;
```

Comment:
Do we really really need to avoid "list" as a variable name. No other way ?

Reply:
Hah :) I believe it is from @afdeprado's changes to avoid same variable names to prevent warnings. Btw, should addressed in this PR